src/couch/priv/couch_js/60/main.cpp

```diff
@@ -432,6 +434,9 @@ main(int argc, const char* argv[])
     JS::CompartmentOptions options;
     JS::RootedObject global(cx, JS_NewGlobalObject(cx, &global_class, nullptr,
                                                    JS::FireOnNewGlobalHook, options));
+
+    JS_SetDefaultLocale(JS_GetRuntime(cx), "en");
```

Comment:
That's the experimental fix, couldn't proove it yet, and the use of `JS_GetRuntime` might not be the right way. Probably a dedicated Runtime-object must be created?